### ISSUE TITLE
Feature Request - Only send updates when needed.

### ISSUE THREAD

**Issue by AvocadoVR** [jsopn/vrc-lyrics#10]
I would like it to update for when I change the song or there is a new lyric that it needs to push. Not updating every 800 spams console and osc. 

There should be two modes.

Update & Update on Song Change & New Lyric.

Updating every so often causes to much spam. It can spam the osc way to much more then what it needs to be called.

**Comment by AvocadoVR** [comment]
Note: Do to this is its a bit unusuable. As it spams to much it timeouts in vrchat and it gets stuck on old lyrics.

**Comment by jsopn** [comment]
![image](https://github.com/jsopn/vrc-lyrics/assets/23272262/7089f5e2-07ab-4360-aa46-5b360eacb062)

For now, while this issue is not complete, you can tweak one of this settings, and set it to value you like.

**Comment by jsopn** [comment]
> > ![image](https://user-images.githubusercontent.com/23272262/265787031-7089f5e2-07ab-4360-aa46-5b360eacb062.png)
> > For now, while this issue is not complete, you can tweak one of this settings, and set it to value you like.
> 
> It I touch that wouldn't that just decrease how often it updates lyrics?

You can decrease a `RateLimit` variable and it will be fine

**Comment by AvocadoVR** [comment]
> > > ![image](https://user-images.githubusercontent.com/23272262/265787031-7089f5e2-07ab-4360-aa46-5b360eacb062.png)
> > > For now, while this issue is not complete, you can tweak one of this settings, and set it to value you like.
> > 
> > 
> > It I touch that wouldn't that just decrease how often it updates lyrics?
> 
> You can decrease a `RateLimit` variable and it will be fine

Yeah. But As I suspected the is going to offset lyrics for songs unless you spam it.

**Comment by AvocadoVR** [comment]
Just want to get an update is the a plan time when this be released.

**Comment by jsopn** [comment]
> Just want to get an update is the a plan time when this be released.
I will try to complete all the planned updates before the end of the month, I unfortunately have a lot of personal stuff going on right now, so I don't have time to get everything done fast enough

**Comment by AvocadoVR** [comment]
That's fine.
